schemas/answers/unit.json

```diff
@@ -45,6 +45,9 @@
       "unit": {
         "$ref": "../lists/units.json#/unit"
       },
+      "unit_length": {
+        "type": "string"
```

Comment:
I don't know how strict we're being about this, but it's probably worth adding a description to this.

Reply:
yeh we should probably make this an enum given there are a fixed set of values

Reply:
Good point, didn't realise there was an enum field.  Added it now